HelpSource/Guides/FFT-Overview.schelp

```diff
@@ -80,28 +80,38 @@ d = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
 d.free;
 ::
 
-Because each PV UGen overwrites the output of the previous one, it is necessary to copy the data to an additional buffer at the desired point in the chain in order to do parallel processing of input without using multiple FFT UGens. link::Classes/PV_Copy:: allows for this.
+As seen below, SuperCollider can create chains of PV processes, which alter the same FFT buffer in series. The PV Ugen writes the output of its process in place, and the effect of one process feeds into the next:
+
 code::
 (
```

Comment:
could you please inline using tabs? In general: documentation and sclang uses tabs, and the c++ side of sc uses spaces.